vowpalwabbit/loss_functions.h

```diff
@@ -10,6 +10,7 @@
 class loss_function
 {
 public:
+  // Information on how to implement a custom loss function in [Loss functions � VowpalWabbit/vowpal_wabbit Wiki (github.com)](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Loss-functions#how-to-write-my-own-loss-function)
```

Comment:
Put this comment at line 10 above the class definition. Maybe no need for markdown?

Reply:
I moved it there and removed the markdown format.